coverity.rst

```diff
@@ -150,4 +150,4 @@ Dakshesh Vyas <scan-admin@coverity.com>
 
 .. _Coverity Connect: http://scan5.coverity.com:8080/
 
-.. _coverity_model.c: https://hg.python.org/cpython/file/tip/Misc/coverity_model.c
+.. _coverity_model.c: https://raw.githubusercontent.com/python/cpython/master/Misc/coverity_model.c
```

Comment:
I think the url should be https://github.com/python/cpython/blob/master/Misc/coverity_model.c

Reply:
I have updated this url